src/worker/ingestion/action-manager.ts

```diff
@@ -37,12 +37,6 @@ export class ActionManager {
         const refetchedAction = await this.db.fetchAction(actionId)
         const wasCachedAlready = teamId in this.actionCache && actionId in this.actionCache[teamId]
         if (refetchedAction) {
-            status.info(
-                '🍿',
-                wasCachedAlready
-                    ? `Refetched action ID ${actionId} (team ID ${teamId}) from DB`
-                    : `Fetched new action ID ${actionId} (team ID ${teamId}) from DB`
-            )
```

Comment:
💯

Reply:
I think we should leave this in, but at a reduced `debug` level, which would only be shown with `LogLevel.Debug` so that normally there's no spam

Reply:
Sounds good to me

Reply:
Actually, involves adding debug to status, so I'll pull it out into a separate PR instead.